core/src/main/java/io/questdb/griffin/engine/AbstractVirtualFunctionRecordCursor.java

```diff
@@ -39,13 +39,8 @@
 
     public AbstractVirtualFunctionRecordCursor(ObjList<Function> functions, boolean supportsRandomAccess) {
         this.functions = functions;
-        if (supportsRandomAccess) {
-            this.recordA = new VirtualRecord(functions);
-            this.recordB = new VirtualRecord(functions);
-        } else {
-            this.recordA = new VirtualRecordNoRowid(functions);
-            this.recordB = null;
-        }
+        this.recordA = new VirtualRecord(functions);
+        this.recordB = new VirtualRecord(functions);
```

Comment:
this was to avoid allocating an object where wasn't necessary, why to change?

Reply:
ok